Makefile

```diff
@@ -29,18 +29,9 @@ GOPKGS		= $(shell go list ./... \
 
 APP_VER		:= $(shell git describe --always 2> /dev/null || echo "unknown")
 
-# linker flags to set build info variables
-BUILD_SYM	:= github.com/envoyproxy/go-control-plane/pkg/version
-LDFLAGS		+= -X $(BUILD_SYM).version=$(APP_VER)
-LDFLAGS		+= -X $(BUILD_SYM).gitRevision=$(shell git rev-parse --short HEAD 2> /dev/null  || echo unknown)
-LDFLAGS		+= -X $(BUILD_SYM).branch=$(shell git rev-parse --abbrev-ref HEAD 2> /dev/null  || echo unknown)
-LDFLAGS		+= -X $(BUILD_SYM).buildUser=$(shell whoami || echo nobody)@$(shell hostname -f || echo builder)
-LDFLAGS		+= -X $(BUILD_SYM).buildDate=$(shell date +%Y-%m-%dT%H:%M:%S%:z)
-LDFLAGS		+= -X $(BUILD_SYM).goVersion=$(word 3,$(shell go version))
-
```

Comment:
why are you removing this? This is needed to build the pkg/version that allows us to see the SHA associated with the binary.

Reply:
This is entirely unnecessary in a library. There is no main function, why do we need ldflags?

Reply:
We can add those, but if this is to be used as part of another project, LD flags make little sense. We can them back with the CLI.

Reply:
in that case, fine.

Reply:
CLIs take a lot of time. And noone likes yet another CLI. It's easy to build a main with this library, where LD flags can be added.

Reply:
👍 Turbine Labs will consume this as a library, and we have our won way of managing injection of this sort of thing. Thanks!